resources/views/seasons/show.blade.php

```diff
@@ -34,6 +34,10 @@ class="btn-osu-big btn-osu-big--rounded-thin"
     <div class="js-react--seasons-show"></div>
 @endsection
 
+@section('scores')
+    @include('seasons._rankings_table', [$divisions, $scores])
```

Comment:
that's not how passing variable works. It should be `['var' => $thing]` to access it as `$var` in the included view. It only seemed to work because all variables from current context are passed through.